frontend/src/scenes/billing/CreditCTAHero.tsx

```diff
@@ -51,18 +79,54 @@ export const CreditCTAHero = (): JSX.Element | null => {
+                            costs more predictable!
                         </p>
                         <LemonButton type="primary" onClick={() => showPurchaseCreditsModal(true)} className="mt-4">
                             Buy credits
                         </LemonButton>
+                        {creditOverview.estimated_monthly_credit_amount_usd > 1667 && (
+                            <>
+                                <LemonDivider className="my-4" />
+                                <div className="mt-2 mb-0 max-w-xl">
+                                    <p className="mb-2">
+                                        Looking for even more? Our enterprise plan is waiting for you ...
+                                    </p>
+                                    <ul className="pl-4">
+                                        <li className="flex gap-2 items-center">
+                                            <IconCheckCircle className="text-success shrink-0" />
+                                            <span>
+                                                Get <strong>customized training</strong> for you and your team
+                                            </span>
+                                        </li>
+                                        <li className="flex gap-2 items-center">
+                                            <IconCheckCircle className="text-success shrink-0" />
+                                            <span>
+                                                Get dedicated support via <strong>private Slack channel</strong>
+                                            </span>
+                                        </li>
+                                        <li className="flex gap-2 items-center">
+                                            <IconCheckCircle className="text-success shrink-0" />
+                                            <span>
+                                                We'll even send you <strong>awesome free merch</strong>
+                                            </span>
+                                        </li>
+                                    </ul>
+                                    <p className="mt-2 mb-0">
+                                        <Link to="mailto:sales@posthog.com?subject=Let's talk enterprise!">
+                                            Talk to sales
+                                        </Link>{' '}
+                                        to learn more.
+                                    </p>
+                                </div>
+                            </>
+                        )}
                     </>
                 )}
             </div>
             {width && width > 500 && (
                 <div className="shrink-0 relative w-50 pt-4 overflow-hidden">
-                    <BlushingHog className="w-50 h-50 -my-5" />
+                    <SurprisedHog className="w-50 h-50 -my-5" />
```

Comment:
I think we can do this to make the hog face the content

```suggestion
                    <SurprisedHog className="w-50 h-50 -my-5 scale-x-[-1]" />
```